src/oscar/apps/catalogue/migrations/0022_auto_20201006_1246.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.10 on 2020-10-06 11:46
+
+from django.db import migrations, models
```

Comment:
Please generate a single migration for all the changes, to replace the two you currently have.